src/components/Stepper/index.tsx

```diff
@@ -104,62 +126,79 @@ const StyledContainer = styled(Box, {
   border: 1px solid ${({ theme }) => theme.colors.gray300};
   border-radius: 4px;
   ${({ disabled, theme }) =>
-    disabled &&
-    `
-    > ${StyledTouchable}, ${StyledInput}, ${StyledCenterTouchable} {
-      ${disabledStyles({ disabled, theme })}
-    }
-  `}
+    disabled
+      ? css`
+          > ${StyledTouchable}, ${StyledInput}, ${StyledCenterTouchable} {
+            ${disabledStyles({ disabled, theme })}
+          }
+        `
+      : ''}
 `
 
-const Stepper = ({
-  disabled,
-  maxValue,
-  minValue,
-  name,
-  onBlur,
+type StepperProps = {
+  disabled?: boolean
+  maxValue?: number
+  minValue?: number
+  name?: string
+  onChange?(data: unknown): void
+  onMaxCrossed?(...args: unknown[]): unknown
+  onMinCrossed?(...args: unknown[]): unknown
+  size?: ContainerSizesType
+  step?: number
+  text?: string
+} & Omit<InputHTMLAttributes<HTMLInputElement>, 'size'>
+
+const Stepper: VoidFunctionComponent<StepperProps> = ({
+  disabled = false,
+  maxValue = 100,
+  minValue = 0,
+  name = 'stepper',
   onChange,
   onFocus,
+  onBlur,
   onMaxCrossed,
   onMinCrossed,
-  size,
-  step,
-  text,
+  size = 'large',
+  step = 1,
+  text = '',
   value,
   ...props
 }) => {
-  const inputRef = useRef()
-  const [inputValue, setInputValue] = useState(parseIntOr(value, minValue))
+  const inputRef =
+    useRef<HTMLInputElement>() as MutableRefObject<HTMLInputElement>
+  const [inputValue, setInputValue] = useState(
+    parseIntOr(value as string, minValue),
```

Comment:
No need to parse when value is already a number no ?

Reply:
Sadly value is `var value: string | number | readonly string[] | undefined`

Reply:
You could do 
```suggestion
    typeof value === "string" ? parseIntOr(value, minValue) : value,
```